k8s-bench/tasks/resize-pvc/verify.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Configuration
+PVC_NAME="storage-pvc" 
+EXPECTED_SIZE="15Gi"
+RETRY_ATTEMPTS=10 
+RETRY_INTERVAL_SECONDS=15 
+
+echo "Attempting to get PV name from PVC: $PVC_NAME"
+
+# Dynamically get the PV name from the PVC
+PV_NAME=$(kubectl get pvc "$PVC_NAME" -n resize-pv -o jsonpath='{.spec.volumeName}')
+
+if [ -z "$PV_NAME" ]; then
+  echo "Error: Could not retrieve PersistentVolume name for PVC '$PVC_NAME'. Make sure the PVC exists and is bound."
+  exit 1
+fi
+
+echo "Found PersistentVolume '$PV_NAME' for PVC '$PVC_NAME'."
+echo "Verifying size of PersistentVolume: $PV_NAME to be $EXPECTED_SIZE"
+
+for i in $(seq 1 $RETRY_ATTEMPTS); do
+  echo "Attempt $i of $RETRY_ATTEMPTS..."
+
+  PV_CAPACITY=$(kubectl get pv "$PV_NAME" -o jsonpath='{.spec.capacity.storage}')
```

Comment:
I wonder if we can rewrite this with kubectl --wait for the capacity condition to be met. See other evals where we use kubectl with wait